src/main/kotlin/com/learnspigot/bot/help/OpenThreadListener.kt

```diff
@@ -0,0 +1,9 @@
+package com.learnspigot.bot.help
+
+import net.dv8tion.jda.api.events.thread.GenericThreadEvent
+import net.dv8tion.jda.api.events.thread.ThreadRevealedEvent
+import net.dv8tion.jda.api.hooks.ListenerAdapter
+
+class OpenThreadListener : ListenerAdapter() {
```

Comment:
Point of this class?

Reply:
Might have been to pin the first message in the thread, ill either delete or add functionality

Reply:
Could you still commit whatever changes for this please

Reply:
> Could you still commit whatever changes for this please

Yeah doing it right now.